build.gradle

```diff
@@ -246,6 +246,7 @@ task fastbuild(dependsOn: [
 
 task travisbuild(dependsOn: [
     fastbuild,
+    ':dbfit-java:db2:integrationTest',
```

Comment:
Light weight because of the provisioning or because of the tests that we're running?